packages/paste-design-tokens/tokens/themes/sendgrid/aliases/color.yml

```diff
@@ -2,4 +2,9 @@ imports:
   - ../../../aliases/color.yml
 aliases:
   curious-blue: "#1A82E2"
+  sg-blue-80: "#489BE8"
```

Comment:
This is more a Paste design team question but wondering if instead of curious-blue we should rename to sg-blue and sg-blue-darker to be consistent with your convention with sg-danger and sg-danger-darker

Reply:
I had the same question!

Reply:
I'm thinking `luminous-fire`, `shady-crimson` `im-blue-a-blah-de-a-blah-da`